### PR TITLE
feat: Configure administration site portal, navigations and pages - EXO-66546 - Meeds-io/MIPs#88

### PR DESCRIPTION
**Pull request by IlhemEssaadi**

### CODE REVIEW

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

File: extension/war/src/main/resources/locale/navigation/portal/administration_en.properties [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (C) 2020 eXo Platform SAS.
```
Comment (azayati) [inline extension/war/src/main/resources/locale/navigation/portal/administration_en.properties]: update header

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/navigation.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+-->
+<node-navigation
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_objects_1_6 http://www.gatein.org/xml/ns/gatein_objects_1_6"
+        xmlns="http://www.gatein.org/xml/ns/gatein_objects_1_6">
+  <priority>3</priority>
+
+  <page-nodes>
```
Comment (azayati) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/navigation.xml:27]: Remove `<visibility>DISPLAYED</visibility>` , it is the default value

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/navigation.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+          <name>profile</name>
+          <label>#{portal.administration.profile}</label>
+          <icon>fas fa-user-cog</icon>
+          <page-reference>portal::administration::profileManagement</page-reference>
+        </node>
+        <node>
+          <name>spaces</name>
+          <label>#{portal.administration.spaces}</label>
+          <icon>fas fa-people-arrows</icon>
+          <page-reference>portal::/platform/users::spacesAdministration</page-reference>
```
Comment (azayati) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/navigation.xml]:
```suggestion
          <page-reference>portal::administration::spacesAdministration</page-reference>
```

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/pages.xml [hunk 4]
```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (azayati) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/pages.xml:1]: update header

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/portal.xml [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+  This file is part of the Meeds project (https://meeds.io/).
+
+  Copyright (C) 2022 Meeds Association contact@meeds.io
```
Comment (azayati) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/portal.xml]: update header

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/portal.xml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  <access-permissions>*:/platform/users</access-permissions>
+  <edit-permission>manager:/platform/administrators</edit-permission>
+  <properties>
+    <entry key="sessionAlive">onDemand</entry>
+    <entry key="showPortletInfo">0</entry>
+  </properties>
+  <portal-layout>
+    <page-body> </page-body>
+  </portal-layout>
+  <displayed>false</displayed>
```
Comment (azayati) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/portal.xml]: put this tag <displayed> after <label> tag

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/social-portal-configuration.xml [hunk 7]
```diff
@@ -231,5 +231,43 @@
...
+    <component-plugin>
+      <name>new.portal.config.user.listener</name>
+      <set-method>initListener</set-method>
+      <type>org.exoplatform.portal.config.NewPortalConfigListener</type>
+      <description>this listener init the portal configuration</description>
+      <init-params>
+        <value-param>
+          <name>override</name>
+          <description>The flag parameter to decide if portal metadata is overriden on restarting server</description>
+         <value> ${exo.social.portalConfig.administration.metadata.override:true} </value>
```
Comment (azayati) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/social-portal-configuration.xml]:
```suggestion
         <value>true</value>
```

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/social-portal-configuration.xml [hunk 8]
```diff
@@ -231,5 +231,43 @@
...
+        </value-param>
+        <object-param>
+          <name>portal.configuration</name>
+          <description>description</description>
```
Comment (azayati) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/social-portal-configuration.xml]:
```suggestion
```

**Review by azayati — CHANGES_REQUESTED** [review]

File: extension/war/src/main/resources/locale/navigation/portal/administration_en.properties (hunk 1 above)
Comment (azayati) [reply extension/war/src/main/resources/locale/navigation/portal/administration_en.properties]: you should declare the file into https://github.com/Meeds-io/social/blob/develop/translations.properties#L64 like done for global site, in order to be considered by crowdin

**Review by azayati — CHANGES_REQUESTED** [review]

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/portal.xml [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+-->
+
+<portal-config>
```
Comment (azayati) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/portal/administration/portal.xml]:
```suggestion
<portal-config
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_objects_1_10 http://www.gatein.org/xml/ns/gatein_objects_1_10"
        xmlns="http://www.gatein.org/xml/ns/gatein_objects_1_10">
```

**Review by azayati — CHANGES_REQUESTED** [review]

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**